src/components/section-navigation/_section-navigation.scss

```diff
@@ -45,6 +45,10 @@
 
     &__item-header {
         @extend .ons-u-mb-no;
+
+        font-size: 1.125rem;
+        font-weight: 700;
+        line-height: 1.6;
```

Comment:
I think a better solution here would be to extend the `ons-u-fs-r--b` class instead.

The font-weight on line 29 and the margin on line 71 can also be removed as they aren't having any effect in this instance